packages/happn-util-crypto/package.json

```diff
@@ -13,19 +13,20 @@
     "test-unit": "mocha test/unit/* --recursive"
   },
   "dependencies": {
-    "happn-commons": "1.2.0",
     "crypto-browserify": "^3.12.0",
+    "happn-commons": "1.2.0",
+    "node-forge": "^1.3.1",
```

Comment:
node-forge adds 1.66 MB unpacked size to the gateway firmware for very limited functionality. Is there a way to achieve this with less code?